rework change to RFC4053 section (#79)
intarchboard/draft-iab-rfc4053bis: draft-iab-rfc4053bis.md

## Changes compared to RFC4053

> This revision of RFC4053 removes all tooling details (multiple section completely,

Reviewer: ```suggestion
This revision of RFC4053 removes all tooling details (multiple sections completely,
```

> large parts of the intro, most of the security consideration and the whole appendix)

Reviewer: ```suggestion
large parts of the intro, most of the security considerations and the whole appendix)
```

and focuses on guidance and requirements.

> It sustains the practical requirements on the content of a liaison statement and its metadata,

Author: ```suggestion
It retains the practical requirements on the content of a liaison statement and its metadata,
```

however, it simplifies the contact information model and removes the "For Comment" purpose.

> Moreover, this revision presents a mayor editorial rework, adds new consideration on

Reviewer: ```suggestion
Moreover, this revision presents a mayor editorial rework, adds new considerations regarding
```

publicly recording liaison statements, integrates approval requirements that where previously

> in RFC4052 instead, and adds consideration on the relation with the IRTF.

Reviewer: ```suggestion
in RFC4052 instead, and explains the relationship with the IRTF.
```

> Strong emphasis is taken, including overview text in the introduction, on the consensus

Reviewer: ```suggestion
Strong emphasis is placed, including overview text in the introduction, on the consensus
```

level of statements and that there is no "special standing" in the IETF process for received statements,
other then providing an easy-to-find, public recording. Further, the process and requirements for sending replies

> has been editorially rewored and clarified with a document structure that should make it easier to find and process

Author: ```suggestion
has been editorially reworked and clarified with a document structure that should make it easier to find and process
```
